app/views/govuk_component/docs/search.yml

```diff
@@ -0,0 +1,19 @@
+name: "Search"
+description: "Search box"
+fixtures:
+  default: {}
+  for_use_on_a_dark_but_not_black_background:
+    on_dark_background: true
+  change_label_text:
+    label_text: "Search all the things"
+  stop_the_label_appearing_inline:
+    inline_label: false
+  apply_custom_styling_to_the_label:
+    inline_label: false
+    label_text: "<span class=\"fixture-title\">Search this website</span>"
```

Comment:
This example is based on a real world use case where I intend the search component to eventually be used (https://www.gov.uk/search?q=test) where the page has a H1 is placed inside a label, but that's invalid, so I changed the example given from an included H1 to a span with the same classes, to simply demonstrate that markup can be included in the label if required. Example updated to be closer to a real world example.